charts/ai-software-templates/chatbot/templates/pvc.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/name: {{ .Release.Name }} 
   name: {{ .Release.Name }}
-  namespace: {{ .Release.Namespace }}
```

Comment:
same why did we remove the namespace setting question

Reply:
I'm closing in favor of https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31#discussion_r1923592267

It will be handled similar to what we decide in this conversation